README.md

```diff
@@ -1,2 +1,216 @@
-# David-Verde-microverse-final-capstone-Back-End
-Final Capstone Project For Microverse Ruby on Rails Module
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] You are using Rspec, but the "test" folder still exists which is used by another test framework.  Kindly delete that folder if you are not using it.  👍

Reply:
Thank you for the advice @Meltrust . We already deleted the test folder.

Reply:
> Thank you for the advice @Meltrust . We already deleted the test folder.

@DarioAlessioR Please notice that I deleted one of my review comments, because all the tests that your wrote are indeed working.  Happy coding.

Reply:
Ok thank you @Meltrust .

Reply:
Hi @Meltrust . We already fixed all issues and tried to ask for a review again in the dashboard, but it shows the status "Review Pending" yet.

Please let us know if we have to do something else to continue.

Thank you again.

Reply:
> Hi @Meltrust . We already fixed all issues and tried to ask for a review again in the dashboard, but it shows the status "Review Pending" yet.
> 
> Please let us know if we have to do something else to continue.
> 
> Thank you again.

Hey @DarioAlessioR, I was still reviewing the frontend.  Normally, the frontend is a lot more time-consuming than the backend.  Kind regards.